.travis.yml

```diff
@@ -1,16 +1,11 @@
 language: dart
-dist: precise
+sudo: required
 dart:
-  - stable
-with_content_shell: true
-before_install:
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3
-script:
-  - pub get --packages-dir
-  - pub run dart_dev format --check
-  - pub run dart_dev analyze
-  - pub run dart_dev test
-  - pub run dart_dev coverage --no-html
-  - bash <(curl -s https://codecov.io/bash) -f coverage/coverage.lcov
+  - 1.24.3
+  - dev
+dart_task:
+  - test: --platform vm
+  - test: --platform chrome
+  - dartanalyzer: lib example
```

Comment:
any reason not to also analyze tool and test directories? And if not, I think you can just leave this as `- dartanalyzer`

Reply:
no reason .. just cause copy/paste. will adjust.